doc/dao/Managing Git & Formatting PRs.md

```diff
@@ -0,0 +1,53 @@
+# Managing Git for Primero
+
+As part of the process for contributing to Primero, it is necessary to maintain a specific workflow for using Git and GitHub.
+
+You should initially create a fork of Primero on GitHub. If you are working on Primero as part of an organization, there might already be an existing fork on GitHub.
+
+![Forking on github!](Attachments/fork.webp)
+Once you have created a fork, you should add your fork as a remote destination to your local copy of the primero code.
+
+Initially, you should create a branch with a descriptive name which identifies the feature you are working on, and push the branch to your fork of Primero hosted on GitHub.
+
+```bash
+git checkout develop
+git pull
+git remote add fork https://github.com/<YOUR GITHUB USERNAME>/primero.git
+git fetch fork
+git checkout -b my-branch-name
+git push --set-upstream fork my-branch-name
+```
+
+## Creating a Pull Request
+
+Once you are ready to have your contribution reviewed by the core development team, you should open a Pull Request on the Primero repository. The development team will then review the contribution, and suggest changes. Once you implement those changes, the team will review the contribution again, and either the review process will continue, or the changes will be approved for merging. Once the changes are merged, and the contribution is concluded, you will be able to ask for payment from the DAO.
```

Comment:
Let's call out that any new PRs must be created against the `develop` branch.

Reply:
Or perhaps `develop-dao`. I think we will need to sort out how this code will be reviewed for UAT/QA